### PR TITLE
Refine

### PR DESCRIPTION
**Pull request by aurexav**
- Manage dependencies with version control
- Improve essay titles
- Automatically detect system theme
- Reduce brightness in dark mode
- Configure extensions/plugins

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
-          - Account System: 'evm/ethereum-compatibility/account-system.md'
-          - JSON-RPC API: 'evm/ethereum-compatibility/jsonrpc.md'
-          - Gas Compatibility: 'evm/ethereum-compatibility/gas.md'
-          - EVM Tracing: 'evm/ethereum-compatibility/evm-tracing.md'
+      - Overview: "msgport/overview.md"
+      - Glossary: "msgport/glossary.md"
+      - Interfaces: "msgport/interfaces.md"
+      - Workflow: "msgport/workflow.md"
+      - API Documentation: "msgport/api.md"
+      - Security Scan: "msgport/scan.md"
```
Comment (hackfisher) [inline mkdocs.yml]:
```suggestion
      - Message Scan: "msgport/scan.md"
```

File: docs/blog/index.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Blog
```
Comment (boundless-forest) [inline docs/blog/index.md]: What's this file for?

File: mkdocs.yml [hunk 3]
```diff
@@ -8,44 +8,50 @@ theme:
   logo: resource/pics/darwinia-logo.png
   favicon: resource/pics/background.png
   font:
-      text: Open Sans
-      code: Roboto Mono
+    text: San Francisco
+    code: SFMono-Regular
```
Comment (boundless-forest) [inline mkdocs.yml:12]: Please don't change the font. 

I don't think the new one is better than the current one. Or you can give me something more convincing.

File: mkdocs.yml [hunk 4]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
+      - Supported Networks: "msgport/networks.md"
+      - Use Cases:
+          - Overview: "msgport/use-cases/overview.md"
+          - Order Clearing: "msgport/use-cases/order-xclearing.md"
+          - XToken Management: "msgport/use-cases/xtoken.md"
+          - XAccount Operations: "msgport/use-cases/xaccount.md"
+          - XDAO Functions: "msgport/use-cases/xdao.md"
+      - Tutorials:
+          - Using Remix on Pangolin > Sepolia: "msgport/tutorial/remix-demo.md"
+          - Script-Based Setup on Pangolin > Sepolia: "msgport/tutorial/script-demo.md"
```
Comment (boundless-forest) [inline mkdocs.yml]: Honestly, I prefer the old one. Looks more neat.

File: mkdocs.yml [hunk 5]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
+      - FAQs: "msgport/faq.md"
+      - Troubleshooting: "msgport/troublesome.md"
+  - EVM+:
+      - Overview: "evm/overview.md"
+      - Chains and Explorers:
```
Comment (boundless-forest) [inline mkdocs.yml:159]: Why not And?

File: mkdocs.yml [hunk 6]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
-              - Awesome Tutorial: 'evm/tutorial/smart-contract/awesome-tutorial.md'
-          - Chain Guide:
-              - Chain Database Snapshot: 'evm/tutorial/chain/snapshot.md'
-              - Run Collator Node: 'evm/tutorial/chain/run-collator-node.md'
-              - Run Archive Node: 'evm/tutorial/chain/run-archive-node.md'
-              - Run EVM Tracing Node: 'evm/tutorial/chain/run-evm-tracing-node.md'
-              - Run Dev Node: 'evm/tutorial/chain/run-dev-node.md'
-              - Run Relay Chain Testnet: 'evm/tutorial/chain/run-relay-chain-testnet.md'
+              - Overview: "evm/ethereum-compatibility/precompiles/overview.md"
+              - Ethereum Contracts: "evm/ethereum-compatibility/precompiles/ethereum.md"
```
Comment (boundless-forest) [inline mkdocs.yml]: No need to add `Contracts`, there is already one in the folder `Precompiled Contracts`.

File: docs/msgport/use-cases/overview.md
Comment (boundless-forest) [inline docs/msgport/use-cases/overview.md:1]: Note: the path change will break some links in the website. Update them later after this merge.

File: mkdocs.yml (hunk 3 above)
Comment (hackfisher) [reply mkdocs.yml:12]: I would vote for new font as it looks more clear in my brower.

image

File: mkdocs.yml (hunk 5 above)
Comment (hackfisher) [reply mkdocs.yml:159]: Refer https://docs.soliditylang.org/en/v0.8.25/

ChatGPT:

> In English titles, conjunctions and prepositions are usually lowercase, except when they are the first word of the title or the last word. Words like "and", "or", "but", "on", "in", "at", and "by" are generally not capitalized in titles unless they appear at the beginning or the end. This is a common rule in English writing for capitalizing titles. If you need the rules for titles in a specific language, please let me know, and I can provide more specific assistance.

File: mkdocs.yml (hunk 6 above)
Comment (hackfisher) [reply mkdocs.yml]: I prefer:
```suggestion
              - Ethereum Precompile: "evm/ethereum-compatibility/precompiles/ethereum.md"
```

File: mkdocs.yml [hunk 7]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
+              - Staking Contracts: "evm/ethereum-compatibility/precompiles/staking.md"
+              - Deposit Contracts: "evm/ethereum-compatibility/precompiles/deposit.md"
+              - Commitment Token Contracts: "evm/ethereum-compatibility/precompiles/commitment-token.md"
+              - USDT Contracts: "evm/ethereum-compatibility/precompiles/usdt.md"
+              - State Storage Contracts: "evm/ethereum-compatibility/precompiles/state-storage.md"
+              - Dispatch Mechanism: "evm/ethereum-compatibility/precompiles/dispatch.md"
+              - Conviction Voting: "evm/ethereum-compatibility/precompiles/conviction-voting.md"
```
Comment (hackfisher) [inline mkdocs.yml]:
```suggestion
              - Ethereum Precompiles: "evm/ethereum-compatibility/precompiles/ethereum.md"
              - Staking Precompile: "evm/ethereum-compatibility/precompiles/staking.md"
              - Deposit Precompile: "evm/ethereum-compatibility/precompiles/deposit.md"
              - KTON Precompile: "evm/ethereum-compatibility/precompiles/commitment-token.md"
              - USDT Precompile: "evm/ethereum-compatibility/precompiles/usdt.md"
              - State Storage Precompile: "evm/ethereum-compatibility/precompiles/state-storage.md"
              - Dispatch Precompile: "evm/ethereum-compatibility/precompiles/dispatch.md"
              - Conviction Voting Precompile: "evm/ethereum-compatibility/precompiles/conviction-voting.md"
```

File: docs/msgport/use-cases/overview.md
Comment (hackfisher) [reply docs/msgport/use-cases/overview.md:1]: https://github.com/darwinia-network/home/pull/410

File: mkdocs.yml (hunk 4 above)
Comment (hackfisher) [inline mkdocs.yml]:
```suggestion
          - Hello Message > Sepolia: "msgport/tutorial/remix-demo.md"
          - Script Message Demo > Sepolia: "msgport/tutorial/script-demo.md"
```

File: mkdocs.yml [hunk 8]
```diff
@@ -67,112 +73,142 @@ extra_javascript:
...
+      - Supported Networks: "msgport/networks.md"
+      - Use Cases:
+          - Overview: "msgport/use-cases/overview.md"
+          - Order Clearing: "msgport/use-cases/order-xclearing.md"
+          - XToken Management: "msgport/use-cases/xtoken.md"
+          - XAccount Operations: "msgport/use-cases/xaccount.md"
+          - XDAO Functions: "msgport/use-cases/xdao.md"
+      - Tutorials:
+          - Hello Message > Sepolia: "msgport/tutorial/remix-demo.md"
+          - Script Message Demo > Sepolia: "msgport/tutorial/script-demo.md"
```
Comment (hackfisher) [inline mkdocs.yml]:
```suggestion
          - Hello Message Using Remix: "msgport/tutorial/remix-demo.md"
          - Message Script Demo: "msgport/tutorial/script-demo.md"
```

**Review by hackfisher — APPROVED** [review]

**Pull request merged**